msg/messages.js

```diff
@@ -109,6 +109,8 @@ Blockly.Msg.HELP = 'Help';
 Blockly.Msg.UNDO = 'Undo';
 /// context menu - Undo the previous undo action.\n{{Identical|Redo}}
 Blockly.Msg.REDO = 'Redo';
+/// context menu - Edit the currently selected function
+Blockly.Msg.EDIT_FUNCTION = 'Edit';
```

Comment:
Should this be `EDIT_PROCEDURE`, I don't think we use "function" much if at all?